test/format/format.h

```diff
@@ -121,6 +127,23 @@ typedef struct {
     bool set;   /* value explicitly set */
 } CONFIGV;
 
+/*
+ * The LANE data structure is used with predictable replay. With predictable replay, we want to make
+ * sure that two threads can never act on the same key. The last bits of the timestamp to be used to
+ * determine a "lane", so it takes a while (LANE_COUNT operations) to cycle through the lanes. A
+ * lane only acts on key numbers whose last bits match the lane. We also keep track of lanes via the
+ * g.lanes array. This guarantees that a lane is only being used one at a time, which in turn
+ * guarantees that a key can only be used once at a time.
+ *
+ * A more complete description of how this fits into predictable replay is in replay.c .
+ */
+typedef struct {
+    uint64_t last_commit_ts;
+    bool in_use;
+} LANE;
+#define LANE_NONE UINT32_MAX /* A lane number guaranteed to be illegal */
+#define LANE_COUNT 1024u
+
```

Comment:
Alphabetize in list of structures. Also please move CONFIG if it can be moved to alphabetize.

Reply:
LANE and CONFIG have been moved.